src/celeritas/ext/g4vg/Converter.cc

```diff
@@ -188,16 +191,36 @@ auto Converter::build_with_daughters(G4LogicalVolume const* mother_g4lv)
         // Convert daughter volume
         VGLogicalVolume* daughter_lv = this->build_with_daughters(g4lv);
 
-        // Use the VGDML reflection factory to place the daughter in the mother
-        // (it must *always* be used, in case parent is reflected)
-        vecgeom::ReflFactory::Instance().Place(
-            (*this->convert_transform_)(g4pv->GetTranslation(),
-                                        g4pv->GetRotation()),
-            vecgeom::Vector3D<double>{1.0, 1.0, flip_z ? -1.0 : 1.0},
-            g4pv->GetName(),
-            daughter_lv,
-            mother_lv,
-            g4pv->GetCopyNo());
+        // extract replication parameters if any
+        int nReplicas = 1;
+        EAxis axis{kUndefined};  // unused
+        bool consuming;  // unused
+        double replicaWidth = 0;  // unused
+        double replicaOffset = 0;  // unused
+        g4pv->GetReplicationData(
+            axis, nReplicas, replicaWidth, replicaOffset, consuming);
+
+        // loop over division replicas (includes trival case of simple
+        // placement)
+        for (int repID = 0; repID < nReplicas; ++repID)
+        {
+            if (nReplicas > 1)
+            {
+                // compute the position/transformation of this replica
+                // (modifies g4pv!!)
+                g4pv->GetParameterisation()->ComputeTransformation(repID, g4pv);
```

Comment:
via this interface all sorts of replicas should be treated generically (tested for divisions and simple cartesian placements)